www/_template/guides/optimize-and-bundle.md

```diff
@@ -56,3 +56,12 @@ Snowpack supports popular bundlers via plugin:
 **For now, we recommend using @snowpack/plugin-webpack until our built-in optimize support is more mature.**
 
 Check out our [Plugins Catalog](/plugins) to browse all available Snowpack plugins, and read the [Plugins Guide](/guides/plugins) if you're interested in creating your own.
+
+### Legacy browser support
+
+You can customize the set of browsers you'd like a bundler to support via the `package.json` "browserslist" property, going all the way back to IE11. This will be picked up when you run `snowpack build` to build for production.
```

Comment:
important to note that this isn't a default thing, and is just the behavior of the Webpack plugin. To make matters more complicated: now that we have built-in `optimize`, we may actually want to document the `optimize.target` option as our built-in legacy browser support.

I have a few things to do to clean up before v3 launch. I can take a stab at documenting this as a part of that, if you're okay with reverting this change.